src/printer.js

```diff
@@ -3005,17 +2931,6 @@ function printFunctionParams(path, print, options, expandArg, printTypeParams) {
     printed = path.map(print, paramsField);
   }
 
-  if (fun.defaults) {
```

Comment:
Looks like an old AST format for default expressions. It was never executed so it might have been from recast?